CHANGELOG.rst

```diff
@@ -9,6 +9,7 @@ v5.14.1
 * Bump certifi from 2022.12.7 to 2023.7.22 in /manager `<https://github.com/lsst-ts/LOVE-manager/pull/198>`_
 * Bump pygments from 2.7.4 to 2.15.0 in /manager `<https://github.com/lsst-ts/LOVE-manager/pull/197>`_
 * Bump cryptography from 41.0.0 to 41.0.2 `<https://github.com/lsst-ts/LOVE-manager/pull/195>`_
+* Add view updates for summit, TTS and BTS `<https://github.com/lsst-ts/LOVE-manager/pull/200>`_
```

Comment:
Please add this record at the beginning of the list in order to have it ordered by descending number of PRs.